playbooks/common/openshift-master/config.yml

```diff
@@ -372,6 +372,12 @@
   - role: flannel_register
     when: openshift.common.use_flannel | bool
 
+- name: Configure Nuage 
+  hosts: oo_first_master
+  roles:
+  - role: nuage_master 
+    when: openshift.common.use_nuage | bool
```

Comment:
In that case, I would suggest moving this role up to the 'Configure master instances' play after the openshift-master role is applied.